src/checkers/inference/util/InferenceUtil.java

```diff
@@ -44,6 +44,14 @@ public static Set<AnnotationMirror> clearAnnos(final AnnotatedTypeMirror atm) {
         return oldAnnos;
     }
 
+    public static void removePrimaryTypeVariableAnnotation(AnnotatedTypeVariable atv, AnnotationMirror potentialVarAnno)  {
```

Comment:
Please add a link to https://github.com/eisop/checker-framework/blob/6f12277290642f8fb89a5c614b31fe419eb0a7b1/framework/src/main/java/org/checkerframework/framework/type/DefaultInferredTypesApplier.java#L134
which does something similar and add a comment that this method should be cleaned up/removed when that code is improved.